lib/Dancer2/Core/Request.pm

```diff
@@ -181,6 +181,18 @@ sub data { $_[0]->{'data'} ||= $_[0]->deserialize() }
 sub deserialize {
     my $self = shift;
 
+    # if a serializer is used (JSON or Mutable) the deserialization of 
```

Comment:
I think the comment describe the problem the code fixes. But without context I am prone to think it describes what the block does. 

Maybe `# don't attempt to deserialize if the form is 'multipart/form-data'`